submit_completion.php

```diff
@@ -35,8 +35,9 @@
 $contextid = required_param('contextid', PARAM_INT);
 $origin    = required_param('lms_origin', PARAM_URL);
 $username  = required_param('student_id', PARAM_RAW_TRIMMED);
+$clientid  = optional_param('client_id', '', PARAM_RAW_TRIMMED);
```

Comment:
This is functional, but I think this should be PARAM_INT

Can you please also bump the version number

Reply:
Hi @brendanheywood ,

Thank you.
Updated the changes.
Please check when you have some time.
Thank you.